src/EditorFeatures/Core.Wpf/InheritanceMargin/MarginGlyph/InheritanceMargin.xaml.cs

```diff
@@ -115,6 +106,15 @@ private void ContextMenu_OnClose(object sender, RoutedEventArgs e)
             ResetBorderToInitialColor();
         }
 
+        private void ContextMenu_OnOpen(object sender, RoutedEventArgs e)
+        {
+            // If this context menu just has one member, then if the context menu open, it means all inheritance targets are shown.
+            if (e.OriginalSource is ContextMenu { DataContext: InheritanceMarginViewModel { HasMultipleMembers: false }})
+            {
+                Logger.Log(FunctionId.InheritanceMargin_TargetsMenuOpen);
```

Comment:
This does not send telemetry, and is only captured in ETW. https://github.com/dotnet/roslyn/issues/51790 . You need to do something like this: https://github.com/dotnet/roslyn/blob/6da1274c9d24c2f90a48290394a951b23617f2a3/src/EditorFeatures/CSharp/CompleteStatement/CompleteStatementCommandHandler.cs#L256